cdap-explore/src/test/java/co/cask/cdap/explore/service/ExploreMetadataTestRun.java

```diff
@@ -74,9 +89,13 @@ public void testGetTables() throws Exception {
                           ),
                           Lists.newArrayList(
                             new QueryResult(Lists.<Object>newArrayList(
-                              "", "default", "my_table", "TABLE", "CDAP Dataset")),
+                              "", OTHER_NAMESPACE_DATABASE, "my_table", "TABLE", "CDAP Dataset")),
```

Comment:
use variables for the name instead of a hardcoded string so that it is easier to update if table names change. (incoming PR prepends "dataset_" or "stream_" to table names)